src/util/sample.cpp

```diff
@@ -499,11 +514,21 @@ void SampleUtil::mixStereoToMono(CSAMPLE* pDest, const CSAMPLE* pSrc,
     }
 }
 
+// static
+void SampleUtil::mixStereoToMono(CSAMPLE* pBuffer, SINT numSamples) {
+    const CSAMPLE_GAIN mixScale = CSAMPLE_GAIN_ONE / (CSAMPLE_GAIN_ONE + CSAMPLE_GAIN_ONE);
+    // note: LOOP VECTORIZED
```

Comment:
The comment contradicts your other comments stating that SINT prevents vectorization??

Reply:
We have originally invented SINT to allow vectorizeing.
I have no idea why int is required in the exceptional case.
See my comment at the beginning of this file.